documentation/modules/configuring/proc-config-kafka-connect-user-authorization.adoc

```diff
@@ -5,43 +5,17 @@
 [id='proc-configuring-kafka-connect-user-authorization-{context}']
 = Configuring Kafka Connect user authorization
 
-This procedure describes how to authorize user access to Kafka Connect.
+[role="_abstract"]
+When using authorization in Kafka, a Kafka Connect user requires read/write access to the cluster group and internal topics of Kafka Connect. 
+This procedure outlines how access is granted using `simple` authorization.
 
-When any type of authorization is being used in Kafka, a Kafka Connect user requires read/write access rights to the consumer group
-and the internal topics of Kafka Connect.
+Properties for the Kafka Connect cluster group ID and internal topics are configured by Strimzi by default. 
+Alternatively, you can define them explicitly in the `spec` of the `KafkaConnect` resource. 
+This is useful when xref:con-kafka-connect-mm2-multiple-instances-{context}[configuring Kafka Connect for multiple instances], as the values for the group ID and topics must differ when running multiple Kafka Connect instances.
 
-The properties for the consumer group and internal topics are automatically configured by Strimzi,
-or they can be specified explicitly in the `spec` of the `KafkaConnect` resource.
-
-.Example configuration properties in the `KafkaConnect` resource
```

Comment:
I removed this as we link to the multiple instances section instead. 
Should we also expand this KC authorization config section to include MM2?